### PR TITLE
refactor/acl migration

### PR DESCRIPTION
**Pull request by jannyHou**
connect to https://github.com/strongloop/loopback-next/issues/4753

Please review with "hide white space changes". And you can review by commits.

Refactored the casbin-auth and jwt-auth services into 2 components:

- /components
  - /casbin-authorization
    - /services
      - /...files
  - /jwt-authentication
    - /services
       - /...files

And update docs accordingly.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/migration/auth/example.md [hunk 1]
```diff
@@ -154,24 +154,46 @@ strategy to verify a user's identity. The authentication setup is borrowed from
...
+decide the `principal`'s access later.
+
+To simplify the implementation for readers, we extracted the services and
+bindings into a component in folder `src/component`, it consists of the
+following files:
 
 - creating the jwt authentication strategy to decode the user profile from
   token. See
-  [file src/services/jwt.auth.strategy.ts](https://github.com/strongloop/loopback-next/tree/master/examples/access-control-migration/src/services/jwt.auth.strategy.ts)
+  [file src/component/jwt.auth.strategy.ts](https://github.com/strongloop/loopback-next/tree/master/examples/access-control-migration/src/component/jwt.auth.strategy.ts).
```
Comment (dhmlau) [inline docs/site/migration/auth/example.md]: nitpick: would it be better if:

---------
See file [src/component/jwt.auth.strategy.ts](https://github.com/strongloop/loopback-next/tree/master/examples/access-control-migration/src/component/jwt.auth.strategy.ts)? 

--------



i.e. `file` outside the link. Similarly for the other occurrences.

Comment (dhmlau) [inline docs/site/migration/auth/example.md]: To be consistent with other folder naming convention, wouldn't it be `components` instead of `component`?

Comment (jannyHou) [reply docs/site/migration/auth/example.md]: right, renamed

Comment (jannyHou) [reply docs/site/migration/auth/example.md]: good point, refactored the casbin-auth and jwt-auth services into 2 components:

    /components
        /casbin-authorization
            /services
                /...files
        /jwt-authentication
            /services
                /...files

**Pull request merged**